### DISCUSSION TITLE
Mod customization panel behavior

### DISCUSSION THREAD

**Discussion by Fabep** [ppy/osu#29618] [Ideas] [answered]
The times I've accidentally collapsed the mod customization panel by clicking on it after opening it by hover are in the __quintillions__. However this is only a slight inconvenience the problem I have is the behavior that comes after clicking on it twice which leads to the state changes of:
```cs
// Hover
touchedThisFrame = ModCustomisationPanelState.ExpandedByHover;

// First click
touchedThisFrame = ModCustomisationPanelState.Collapsed;

// Second click 
touchedThisFrame = ModCustomisationPanelState.Expanded;
```

As I move my cursor outside of the panel I expect the panel to close, however it does not.
From my extremely in-depth deduction I've come to the conclusion that this behavior only exists because of people playing the desktop version of the game using a touch device. 

Therefore my suggestion is to add a setting to either disable the hover event of the Mod Customization Panel, or add a setting to set the state to ExpandedByHover instead of Expanded after the user clicks the panel a second time.

**Comment by peppy** [comment, marked as answer]
We discussed elsewhere just removing the click-to-close completely. ie. click just becomes a noop. You're welcome to PR a change in this direction and we'll take it from there.
